### PR TITLE
staged-recipes is not mandatory for core members

### PR DESCRIPTION
**Pull request by ocefpaf**
@scopatz can you review this one?

### CODE REVIEW

File: src/governance.rst [hunk 1]
```diff
@@ -88,9 +88,8 @@ the time when a vote is called, but may never be less than 24 hrs.
 
 :Nominate new member of core: The proposer must provide
     a sufficient justification as to why the nominee should be welcomed
-    into core. It is necessary that all nominees are current and active
-    members of staged-recipes. Nominees to core must have been members
-    of staged-recipes for at least 1 month (4 weeks) prior to this vote.
+    into core. It is not necessary that all nominees are active
```
Comment (CJ-Wright) [inline src/governance.rst]: Maybe we could state this more generically? 

eg
`Prior service to the community, including but not limited to: serving as a staged-recipes reviewer, working on critical conda-forge infrastructure, and helping to bridge disparate communities are an important part of the nomination process`.

**Comment by scopatz** [comment]
Thanks for putting this in @ocefpaf! I like @CJ-Wright's wording a bit better.

File: src/governance.rst (hunk 1 above)
Comment (ocefpaf) [reply src/governance.rst]: I guess that "a sufficient justification a" is better than listing all possible reasons to justify the invitation and then modifying it when we find an issue, like this one.

**Comment by ocefpaf** [comment]
> I like @CJ-Wright's wording a bit better.

I don't b/c I'm concerned that people will focus on that instead of using common sense. Hopefully the "including but not limited to" will prevent that from happening.

**Comment by scopatz** [comment]
Thanks @ocefpaf @CJ-Wright!

**Pull request merged**